.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
Some of us don't use vscode, so it seems inappropriate to put editor-specific settings into the repo. 
you will most likely need to run `git rm --cached` on the file to remove it from [git](https://git-scm.com/docs/git-rm#git-rm---cached)